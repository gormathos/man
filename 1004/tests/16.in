10
-6.59795 -1.04729 7.06184 5.89354 -9.1139 9.94927 -1.57157 -6.80305 -3.24331 -6.89764
10
1.25962 -3.15317 -5.31772 9.89762 4.70378 -6.12645 2.5672 -6.7904 -9.38641 5.40331
