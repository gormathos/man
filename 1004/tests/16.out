1.00000 12.37006 -132.55687 -2209.11946 893.47050 112481.62415 327255.21381 -1394290.94774 -8161965.41256 -12729818.26868 -6237232.59191
-36836424.40422 -102013.90151 801968.42325 -85098409.81366 -87991743.23726 124931.89238 -91087767.23337 -189.67998 9504855.90251 -37615503.82710
-35945818.44902 -1091657.78581 907890.59765 1554867674.37988 58268319.37659 530436.18879 -39977944.61265 -15696.41040 -51380695.98184 80245246.83348
